src/test/record/records_service/test_response_time.py

```diff
@@ -220,26 +151,10 @@ def test_double_flow_cross_case(self):
         records = create_records(records_raw, columns)
         response = ResponseTime(records)
 
-        expect_raw = [
-            {'start': 0, 'end': 3},
-            {'start': 2, 'end': 3}
-        ]
-        result = to_dict(response.to_records())
-        assert result == expect_raw
-
         expect_raw = [
             {'start_min': 0, 'start_max': 2, 'end': 3},
         ]
-        result = to_dict(response.to_response_records())
-        assert result == expect_raw
-
-        expect_raw = [
-            {'start': 0, 'end': 3},
-            {'start': 0, 'end': 5},
-            {'start': 2, 'end': 3},
-        ]
-
-        result = to_dict(response.to_records(all_pattern=True))
+        result = to_dict(response.to_stacked_bar())
         assert result == expect_raw
```

Comment:
In this test case, it seems not to consider the case: 'end' : 5 in line 238.
Is this OK?

Reply:
The `cross_case` may be different from the desired specification. (The implement may wrong.)

However, as this PR is a refactoring, this investigation and correction will not be conducted.